extensions/interactions/base-interaction-validation.service.ts

```diff
@@ -27,8 +27,8 @@ import { Outcome } from
 import { AppConstants } from 'app.constants';
 
 export interface Warning {
-  type: string;
-  message: string
+  type: string | null;
+  message: string | null;
```

Comment:
Why can this be `null`?

Reply:
Sorry for that. Actually, By Mistake, I confused them with some variables in other files I was looking. I have removed them.